### ISSUE TITLE
[Feauture Request] Adding the ability to manage Auto Update option through a specific URL Scheme

### ISSUE THREAD

**Issue by neerro** [SagerNet/sing-box#2140]
### Operating system

Android

### System version

anyVersion & anyTypeOfDevices

### Installation type

Others

### Description

Dear @nekohasekai,

I would like to propose an enhancement to the application’s functionality that will allow users to manage auto-update parameters when importing profiles via remote links in the format sing-box://import-remote-profile?url=urlEncodedURL#urlEncodedName.

Currently, the ability to modify auto-update parameters (enable/disable and interval) is only available through the user interface. This limits user flexibility as they cannot predefine these parameters in the link.

Proposed Changes:
Support for parameters in the URL:

Implement the ability to pass auto-update parameters through the URL. For example:
**Auto Update Enabled**
sing-box://import-remote-profile?url=urlEncodedURL&autoUpdate=true&autoUpdateInterval=60#urlEncodedName
or it may be leave as it is: sing-box://import-remote-profile?url=urlEncodedURL#urlEncodedName.
**Auto UpdateDisabled**
sing-box://import-remote-profile?url=urlEncodedURL&autoUpdate=false&autoUpdateInterval=0#urlEncodedName

Modifying profile handling logic:

In the method that handles profile imports, add logic to extract the autoUpdate and interval parameters from the URL.
Set the corresponding values in the TypedProfile object.

This will add flexibility and simplify the use of the application for users who prefer to set up their profiles in advance.

### Reproduction

There is no error. It is a Feauture Request.

### Supporter

- [ ] I am a [sponsor](https://github.com/sponsors/nekohasekai/)

### Integrity requirements

- [X] I confirm that I have read the documentation, understand the meaning of all the configuration items I wrote, and did not pile up seemingly useful options or default values.
- [X] I confirm that I have provided the server and client configuration files and process that can be reproduced locally, instead of a complicated client configuration file that has been stripped of sensitive data.
- [X] I confirm that I have provided the simplest configuration that can be used to reproduce the error I reported, instead of depending on remote servers, TUN, graphical interface clients, or other closed-source software.
- [X] I confirm that I have provided the complete configuration files and logs, rather than just providing parts I think are useful out of confidence in my own intelligence.

**Comment by ahlixinjie** [comment]
agreed. now only graphic client implement this. Also need this for bin

**Comment by neerro** [comment]
One year later... Did nothing!

**Comment by Gemini-NX** [comment]
Got a little question, I'm using sing-box with latest version on an Iphone through App stroe TestFlight. The "Auto Update Interval" was set to 3 mins. But I didn't see the profile being auto updated whatever I started related sing-box config or not. Is there any thing I missed to check?
